lib/inferno/config/boot/suites.rb

```diff
@@ -25,5 +25,16 @@
     end
 
     ObjectSpace.each_object(TracePoint, &:disable)
+
+    Inferno::Entities::TestSuite.descendants.each do |descendant|
+      if descendant.id.blank?
+        raise StandardError, 'Error initializing test suites: custom test suite ID cannot be blank'
+      end
+      # When ID not assigned in custom test suites, Runnable.id will return default ID
+      # equal to the custom test suite's parent class name
+      if descendant.id == 'Inferno::Entities::TestSuite'
+        raise StandardError, 'Error initializing test suites: custom test suite ID is not set'
```

Comment:
We need to include the test suite name so that the user knows which suite needs to be fixed.

```suggestion
        raise StandardError, 'Error initializing test suite #{descendant.name}: test suite ID is not set'
```